src/content/self-hosted/manage/air-gapped.mdx

```diff
@@ -107,4 +107,18 @@ Run the following command to install Okteto for the first time or to upgrade an
 </CodeBlock>
 
 
-_Need help with your installation? [Contact us](https://www.okteto.com/contact/) for expert guidance._
\ No newline at end of file
+_Need help with your installation? [Contact us](https://www.okteto.com/contact/) for expert guidance._
+
+## Step 5: Download the Syncthing Binary
```

Comment:
I suggest that we explain that okteto needs the cli and syncthing, where to download them from and where to install them.  [I wrote this in case it helps.](https://github.com/okteto/airgap-installer/blob/main/README.md#how-to-install-okteto-cli-on-an-air-gapped-environment)

```suggestion
## Step 5: Download the client binaries
```